configuration.example.yaml

```diff
@@ -19,6 +19,8 @@ server:
     # certfile: ./poseidon.crt
     # The path to the key file used for TLS
     # keyfile: ./poseidon.key
+    # The path to the ca certificate used for TLS requests.
+    # cafile: ./ca.crt
```

Comment:
I would recommend to keep the same order as we have for `nomad.tls`. There, the `cafile` proceeds the `certfile` and `keyfile`.